.github/labeler.yml

```diff
@@ -1,10 +1,10 @@
-CI:
+type/ci:
```

Comment:
curiosity: would this improve automatic labeling of PRs / newly created issues? Or is this more for internal naming consistency?

Reply:
This is for the automatic labeling of PR's, not for issues as they don't have any files to base the label on.